rfc9204.md

```diff
@@ -1200,12 +1200,13 @@ This is possible even over the Transport Layer Security Protocol
 while TLS and QUIC provide confidentiality protection for content, they only
 provide a limited amount of protection for the length of that content.
 
-Note:
-
-: Padding schemes only provide limited protection against an attacker with these
-  capabilities, potentially only forcing an increased number of guesses to learn
-  the length associated with a given guess. Padding schemes also work directly
-  against compression by increasing the number of bits that are transmitted.
+<aside><t>
+Note: Padding schemes only provide limited protection against an attacker with
+these capabilities, potentially only forcing an increased number of guesses to
+learn the length associated with a given guess. Padding schemes also work
+directly against compression by increasing the number of bits that are
+transmitted.
+</t></aside>
```

Comment:
I couldn't find a kramdown way to do this, so I just inlined the XML.

Reply:
Carsten mentioned that you can use `{:aside}` in another thread, but I don't know exactly how to do that, maybe...

```suggestion
Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.
{:aside}
```

Reply:
:aside and :quote are meant to go with blockquotes:

{:aside}
> Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.

Reply:
Ah, I see.  Thank you!
```suggestion
{:aside}
> Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.
```